src/Search.js

```diff
@@ -0,0 +1,120 @@
+// downshift applies htmlFor for the label, but eslint doesn't recognize that
+// also disabling the specific line causes some weirdness when running lint for syntax
+/* eslint-disable jsx-a11y/label-has-for */
+import React, {useState} from 'react'
+import Router from 'next/router'
+import lunr from 'lunr'
+import {Relative, Box, Text} from '@primer/components'
+import documents from '../searchIndex'
+import SearchItem from './SearchItem'
+import Downshift from 'downshift'
+import SearchInput from './SearchInput'
+import SearchResults from './SearchResults'
+
+const generateBreadcrumb = path => {
```

Comment:
This feels rickety 😬 The other option I considered was requiring a "readable" path value in the front matter, but I don't want to add too many required yaml values to make this Header work, so settled for this

Reply:
Right, I don't actually want a NodeLink here though, it should just be text